packages/create-vite/template-preact-ts/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + Preact + TS</title>
+    <title>{{PROJECT_NAME}}</title>
```

Comment:
Keep all `<title>` as it was before the change. Update it using regex to match `<title>` to `</title>`